ggshield/cmd/utils/hooks.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from ggshield.core.text_utils import display_error
+
+
+def check_user_requested_skip() -> bool:
+    if get_skip_env_var():
+        return True
+    return False
+
+
+def get_skip_env_var() -> bool:
```

Comment:
I think you can merge these two, unless `check_user_requested_skip()` becomes responsible for showing the message.